docs/sphinx/source/index.rst

```diff
@@ -37,8 +37,10 @@ help updating them!
 Please see our 
 `PVSC 2014 paper <http://energy.sandia.gov/wp/wp-content/gallery/uploads/PV_LIB_Python_final_SAND2014-18444C.pdf>`_
 and
-`PVSC 2015 abstract <https://github.com/UARENForecasting/pvlib-pvsc2015/blob/master/pvlib_pvsc_42.pdf?raw=true>`_ 
-for more information.
+`PVSC 2015 abstract <http://energy.sandia.gov/about/publications/download-info/pvlib-python-2015/>`_ 
```

Comment:
This is not the correct link and I should have looked at this more closely before merging. I fixed this with commit 263c596